docs/multicluster/getting-started.md

```diff
@@ -0,0 +1,417 @@
+kind: ClusterClaim
+metadata:
+  name: north-membercluster-id
+  namespace: kube-system
+name: id.k8s.io
+value: test-cluster-north
+---
+apiVersion: multicluster.crd.antrea.io/v1alpha1
+kind: ClusterClaim
+metadata:
+  name: clusterset-id
+  namespace: kube-system
+name: clusterSet.k8s.io
+value: test-clusterset
+---
+apiVersion: multicluster.crd.antrea.io/v1alpha1
+kind: ClusterSet
+metadata:
+    name: test-clusterset
+    namespace: kube-system
+spec:
+    leaders:
+      - clusterID: test-cluster-north
+        secret: "member-north-access-token"
+        server: "https://172.18.0.2:6443"
+    members:
+      - clusterID: test-cluster-north
+    namespace: antrea-mcs-ns
+```
+
+Then update the ClusterSet `test-clusterset` definition as below to include itself
+as member clusters in the namespace `antrea-mcs-ns` where the leader controller is
+running in the cluster `test-cluster-north`.
+
+```yaml
+apiVersion: multicluster.crd.antrea.io/v1alpha1
+kind: ClusterSet
+metadata:
+    name: test-clusterset
+    namespace: antrea-mcs-ns
+spec:
+    leaders:
+      - clusterID: test-cluster-north
+    members:
+      - clusterID: test-cluster-east
+        serviceAccount: "member-east-access-sa"
+      - clusterID: test-cluster-west
+        serviceAccount: "member-west-access-sa"
+      - clusterID: test-cluster-north
+        serviceAccount: "member-north-access-sa"
+    namespace: antrea-mcs-ns
+```
+
+## Use Multi-cluster Custom Resource
+
+After you set up a cluster set properly, you can simply add a `ServiceExport` resource
+as below to export a `Service` from one member cluster to other members in the cluster
+set, you can update the name and namespace according to your local K8s Service.
+
+```yaml
+apiVersion: multicluster.x-k8s.io/v1alpha1
+kind: ServiceExport
+metadata:
+  name: nginx
+  namespace: kube-system
+```
+
+For example, once you export the `kube-system/nginx` Service in the member cluster
+`test-cluster-west`, Antrea multi-cluster controller in the member cluster will create
+two corresponding `ResourceExport` in the leader cluster, and the controller in leader
+cluster will do some computations and create two `ResourceImport` contains all exported
+Service and Endpoints' information. you can check resources as below in leader cluster:
+
+```sh
+$kubectl get resourceexport
+NAME                                        AGE
+test-cluster-west-default-nginx-endpoints   7s
+test-cluster-west-default-nginx-service     7s
+
+$kubectl get resourceimport
+NAME                      AGE
+default-nginx-endpoints   99s
+default-nginx-service     99s
+```
+
+then you can go to the member cluster `test-cluster-east` to check new created
+Service, Endpoints with name `kube-system/antrea-mc-nginx` and a ServiceImport named
+`kube-system/nginx` by multi-cluster controller. If there is already an existing Service
+in the member cluster `test-cluster-east` named `nginx` in namespace `kube-system` without
```

Comment:
Namespace

Reply:
Maybe "an existing Service created by users ..., which should not have the Antrea Multi-cluster annotation"